logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/UserConfigRepository.kt

```diff
@@ -16,6 +16,8 @@ interface UserConfigRepository {
     fun getClassifiedDomainsStatus(): Flow<Either<StorageFailure, ClassifiedDomainsStatus>>
     fun isMLSEnabled(): Either<StorageFailure, Boolean>
     fun setMLSEnabled(enabled: Boolean): Either<StorageFailure, Unit>
+    fun setConferenceCallingEnabled(enabled: Boolean): Either<StorageFailure, Unit>
+    fun isConferenceCallingEnabledFlow(): Flow<Either<StorageFailure, Boolean>>
```

Comment:
Do we need to observe the flag ? checking before starting a call isn't enough ?

Reply:
Nevermind. After actually having a good look of the code in junction with AR again it does seem more logical not having the observe/flow from the useCase and rather as you mentioned, returning it directly as a solely Boolean.